src/coreclr/jit/llvm.h

```diff
@@ -350,19 +342,38 @@ class Llvm
     GenTree* normalizeStructUse(LIR::Use& use, ClassLayout* layout);
 
     unsigned representAsLclVar(LIR::Use& use);
-    GenTree* createStoreNode(var_types nodeType, GenTree* addr, GenTree* data);
-    GenTree* createShadowStackStoreNode(var_types storeType, GenTree* addr, GenTree* data);
     GenTree* insertShadowStackAddr(GenTree* insertBefore, ssize_t offset, unsigned shadowStackLclNum);
 
+    unsigned getCatchArgOffset() const;
+
+    // ================================================================================================================
+    // |                                           Shadow stack allocation                                            |
+    // ================================================================================================================
+public:
+    void Allocate();
+
```

Comment:
I expect that this code structure is more or less temporary and LSSA proper will be put into its own class.